SimG4Core/PhysicsLists/src/CMSMonopolePhysics.cc

```diff
@@ -110,9 +109,9 @@ void CMSMonopolePhysics::ConstructProcess() {
         ph->RegisterProcess(hioni, mpl);
       }
       if (magn != 0.0) {
-        CMSmplIonisation* mplioni = new CMSmplIonisation(magn);
+        G4mplIonisation* mplioni = new G4mplIonisation(magn);
```

Comment:
@civanch could these `new`-s be avoided? Is `mplioni` deleted later?

Reply:
No, it is Geant4 which takes care for object ownership.

Reply:
Great, thanks for the info!